internal/sirius/replace_deputy_document.go

```diff
@@ -0,0 +1,73 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/model"
+	"mime/multipart"
+	"net/http"
+)
+
+func (c *Client) ReplaceDocument(ctx Context, file multipart.File, filename, documentType, direction, date, notes string, deputyId, documentId int) error {
+	var body bytes.Buffer
+
+	source, err := EncodeFileToBase64(file)
```

Comment:
It might be worth moving this function out of add_document now it's used in several places? Maybe we could move it to util, in case we deal with more files in the future

Reply:
I agree moving it out but could we look into moving it to the model/document.go file as putting it in the util is a bit of an anti pattern  
Something like 

On the model
func EncodeFileToBase64(file multipart.File) (string, error) {
	defer file.Close()

	buf := bytes.NewBuffer(nil)
	if _, err := io.Copy(buf, file); err != nil {
		return "", err
	}

	file.Close()

	return base64.StdEncoding.EncodeToString(buf.Bytes()), nil
}

and called like 
model.EncodeFileToBase64(file)